tools/disable-root-account

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+img_type=$(cat /etc/bios-release.json | grep osimage-img-type)
+if [[ $img_type = *"devel"* ]]
+then
+   #only for deploy
+   exit 1
+fi
+
+if [ $# -ne 0 ] && [ $# -ne 2 ]
+then
+    #error
+    exit 1
+fi
+
+if [ $# -eq 2 ] && [ $1 == "-t" ]
+then
+    sleep $2m
```

Comment:
Better `${2}m` or `"$2"m` to make sure a shell interpreter won't look for a variable named `2m` and barf with invalid token.

Reply:
Also, I believe before the sleep there should be a (currently commented-away) usermod to enable the account shell?